homeassistant/components/hue/bridge.py

```diff
@@ -87,14 +97,26 @@ def host(self):
         hass.async_create_task(
             hass.config_entries.async_forward_entry_setup(self.config_entry, "light")
         )
-        hass.async_create_task(
-            hass.config_entries.async_forward_entry_setup(
-                self.config_entry, "binary_sensor"
+
+        if self.add_sensors:
```

Comment:
This PR should have been two PRs. One to add the options flow and one to add remotes.

I don't see the use of allowing to include sensors yes/no. If people really don't want to see it, they can disable the entities but they are not "in the way". 

What would be a use case to disable the sensors?

Reply:
> What would be a use case to disable the sensors?

As exposed in the long answer, I see 2 questions/decisions:
- Disable 1 entity --> forget about 1 item
- Disable 1 platform --> forget about 1 kind of items

In fact, the general idea was to _select_ the kind of devices you want to add/setup/handle